src/main/java/cat/udl/eps/softarch/demo/domain/Take.java

```diff
@@ -17,6 +17,10 @@ public class Take extends Batch {
     @Id
     private Long id;
     private ZonedDateTime lastUpdate;
-
+    @OneToOne
+    private Request request;
```

Comment:
No need to model both sides of the relationship, as per de diagram we are currently modelling the other side. If we model both, we will need to take care of having both in sync

Reply:
Perfect, I take it into account.